python/tutorials/06-fused-attention.py

```diff
@@ -441,9 +441,12 @@ def forward(ctx, q, k, v, causal, sm_scale, split_kernel=False):
         if torch.version.hip is None:
             BLOCK_N = 64 if Lk <= 64 else 32
             num_stages = 4 if Lk <= 64 else 3
+            waves_per_eu = 0 # This is ROCM specific parameter
```

Comment:
I think we don't need to expose this variable on the nv path.

Reply:
I think this part is fine. What Jason brought up about interface changes is more important for now.